Add tabCapture sample (#933)
GoogleChrome/chrome-extensions-samples: api-samples/tabCapture/README.md

# chrome.tabCapture

> A sample demonstrates using the [`chrome.tabCapture`](https://developer.chrome.com/docs/extensions/reference/tabCapture/) API.

Reviewer: ```suggestion
A sample that demonstrates how to use the [`chrome.tabCapture`](https://developer.chrome.com/docs/extensions/reference/tabCapture/) API.
```

## Overview

> In this sample, the `chrome.tabCapture` API is used to capture the contents of the active tab. The captured stream will be displayed in a newly created tab.

Reviewer 2: ```suggestion
In this sample, the `chrome.tabCapture` API is used to capture the contents of the active tab. The captured stream will be displayed in a newly created window.
```

> In this sample, the `chrome.tabCapture` API is used to capture the contents of the active tab. The captured stream will be displayed in a newly created window.

Reviewer: ```suggestion
In this sample, the `chrome.tabCapture` API captures the contents of the active tab. The captured stream is displayed in a new window.
```

## Implementation Notes

> To specify capturing specific tabs, [`tabCapture.getMediaStreamId`](https://developer.chrome.com/docs/extensions/reference/tabCapture/#method-getMediaStreamId) is used.

Reviewer: ```suggestion
Use [`tabCapture.getMediaStreamId`](https://developer.chrome.com/docs/extensions/reference/tabCapture/#method-getMediaStreamId) to capture specific tabs.
```

The `targetTabId` and `consumerTabId` are obtained in the Service Worker, and then passed to the receiver page through the [`tabs.sendMessage`](https://developer.chrome.com/docs/extensions/reference/tabs/#method-sendMessage) method.

> You can find a more detailed implementation guide on audio recording and screen capture at <https://developer.chrome.com/docs/extensions/mv3/screen_capture/#audio-and-video>.

Reviewer: ```suggestion
See the [Audio recording and Screen capture guide](https://developer.chrome.com/docs/extensions/mv3/screen_capture/#audio-and-video) for a more detailed implementation.
```

Reviewer: Can we also add a new section with instructions on how to test the extension?

```
## Running this extension

1. Clone this repository.
2. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
3. _Add the rest of the instructions here_
```
